### PR TITLE
Update README install instructions back to go get.

### PR DESCRIPTION
**Pull request by thodges-gh**
Tested on a fresh build, we can install using `go get`. One caveat is the machine will need `gcc` installed in order for go-ethereum/crypto/secp256k1 to build. I think I'll gather all the required dependencies for a development environment and put it in the Developer Tips wiki page.

### CODE REVIEW

**Comment by dimroc** [comment]
`go get` will **download and install** a package. It works at the moment because we've updated all of our dependencies to be the latest and are therefore not behind some breaking change. If, however, a breaking change happens tomorrow in dependency *xyz*, `go get` will fail. This is what fooled us the first time.

In order to **just download** the package and dependencies, we can do `go get -d ...`. So your instruction for 3) should at the very least should say "Download Chainlink: `go get -d ...`"

Sample:

1. [Install Go 1.9+](https://golang.org/doc/install#install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
2. Install [dep](https://github.com/golang/dep#installation): `$ brew install dep` <br> or `$ go get -u github.com/golang/dep/cmd/dep`
3. `$ go get -d github.com/smartcontractkit/chainlink` (only download Chainlink)
4. `$ cd $GOPATH/src/github.com/smartcontractkit/chainlink`
5. `$ make install`
6. `$ chainlink help`

More reading here: https://stackoverflow.com/questions/24878737/what-is-the-difference-between-go-get-and-go-install

**Comment by thodges-gh** [comment]
Thanks @dimroc, this is exactly what I needed to know. I'm going to run through the steps with -d one more time to test and commit the updated readme.

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Examples of how to utilize and integrate Chainlinks can be found in our [Hello C
 
 1. [Install Go 1.9+](https://golang.org/doc/install#install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
 2. Install [dep](https://github.com/golang/dep#installation): `$ brew install dep` <br> or `$ go get -u github.com/golang/dep/cmd/dep`
-3. Clone ChainLink: `$ git clone https://github.com/smartcontractkit/chainlink.git`
-4. Install the project: `$ make install`
+3. Download ChainLink: `$ go get -d github.com/smartcontractkit/chainlink`
+4. Install: `$ make install` in the `chainlink/` folder
```
Comment (dimroc) [inline README.md]: I would expand `chainlink/` to be the proper path: `$GOPATH/src/github.com/smartcontractkit/chainlink`

**Comment by thodges-gh** [comment]
Okay, I think I found a good trade-off between keeping the install steps few, while having them detailed enough to be followed line-by-line. I'm kind of cheating on step 3 by doing two actions in one, but I'd like to avoid adding a bunch of steps because then it will just look like the Development Setup section.

**Pull request merged**